react-native-libraries.json

```
@@ -11221,5 +11221,19 @@
     "npmPkg": "@okta/okta-react-native",
     "android": true,
     "ios": true
+  },
+  {
+   "githubUrl": "https://github.com/gev2002/react-native-vision-camera-text-recognition",
+    "npmPkg": "react-native-vision-camera-text-recognition",
+    "android": true,
+    "ios": true,
+    "expoGo": true
+  },
+    {
+   "githubUrl": "https://github.com/gev2002/react-native-vision-camera-image-labeler",
+    "npmPkg": "react-native-vision-camera-image-labeler",
+    "android": true,
+    "ios": true,
+    "expoGo": true
```

```suggestion
    "ios": true
  },
    {
   "githubUrl": "https://github.com/gev2002/react-native-vision-camera-image-labeler",
    "npmPkg": "react-native-vision-camera-image-labeler",
    "android": true,
    "ios": true
```

Vision Camera is not supported in Expo Go, thus the libraries depending on it cannot support Go neither.